src/app/commandlineparser.cpp

```diff
@@ -262,6 +264,10 @@ void CommandLineParser::parse(int argc, char** argv)
         m_audioPluginRegistration.failCode = !args1.empty() ? args1[0].toInt() : -1;
     }
 
+    if (m_parser.isSet("audioDelayCompensate")) {
+        m_audioDelayCompensate = m_parser.value("audioDelayCompensate").toInt();
```

Comment:
Did you perhaps intend the following?
```suggestion
        m_options.audio.audioDelayCompensate = m_parser.value("audioDelayCompensate").toInt();
```
(and delete `m_audioDelayCompensate` from commandlineparser.h)

Also, you might want to utilise `intValue` here (see around line 183).